src/integrationTest/java/com/mongodb/spark/sql/connector/MongoSparkConnectorTest.java

```diff
@@ -16,27 +16,134 @@
  */
 package com.mongodb.spark.sql.connector;
 
+import static java.util.Arrays.asList;
+import static org.apache.spark.sql.types.DataTypes.createStructField;
+import static org.apache.spark.sql.types.DataTypes.createStructType;
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertIterableEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
-import java.util.ServiceConfigurationError;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
 
+import org.apache.spark.sql.AnalysisException;
+import org.apache.spark.sql.DataFrameWriter;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
 import org.apache.spark.sql.SparkSession;
-import org.junit.jupiter.api.DisplayName;
+import org.apache.spark.sql.streaming.StreamingQuery;
+import org.apache.spark.sql.types.DataTypes;
+import org.apache.spark.sql.types.StructType;
 import org.junit.jupiter.api.Test;
 
+import org.bson.BsonDocument;
+
+import com.mongodb.client.model.Projections;
+
+import com.mongodb.spark.sql.connector.config.MongoConfig;
 import com.mongodb.spark.sql.connector.mongodb.MongoSparkConnectorTestCase;
 
 class MongoSparkConnectorTest extends MongoSparkConnectorTestCase {
+  private static final String RESOURCES_JSON = "src/integrationTest/resources/json/";
+
+  @Test
+  void testReadAreUnsupported() {
+    SparkSession spark = getOrCreateSparkSession();
+    assertThrows(UnsupportedOperationException.class, () -> spark.read().format("mongodb").load());
+  }
+
+  @Test
+  void testSupportedWriteModes() {
+    SparkSession spark = getOrCreateSparkSession();
+
+    Dataset<Row> df = spark.read().json("src/integrationTest/resources/json/people.json");
```

Comment:
`RESOURCES_JSON` was probably intended to be used here before the `people.json` part.

Reply:
Done.

Reply:
[optional]
Note that the new code is not equivalent to the previous one. `RESOURCES_JSON + people.json` is equivalent. Not sure if this is important, so pointing out just in case.

Reply:
Not important in this instance - so has been simplified.